lightly/cli/_helpers.py

```diff
@@ -51,6 +54,18 @@ def fix_input_path(path):
     return path
 
 
+def fix_hydra_arguments(config: str = 'config'):
+    """Helper to make hydra arugments adaptive to installed hydra version
+    
+    Hydra introduced the `version_base` argument in version 1.2.0
+    We use this helper to provide backwards compatibility to older hydra verisons.    
+    """
+    if version_compare(hydra.__version__, '1.1.2') > 0:
```

Comment:
This `version_compare` is only safe if the version is of the format `X.Y.Z` and this is fine for our own version checks atm. However, we have no control over hydra versions (they could release `1.2.3beta` for example and I don't know how the version check would handle this. So my suggestion would be to wrap the whole thing in a try-catch and return some save default value.